### PR TITLE
Install compas_ghpython for Rhino 6.0 for Mac by default

### PR DESCRIPTION
**Pull request by tetov**
I think this was actually mentioned at the ITA19 course by @brgcode. Troubleshot a friend's compas install on Mac and realized that compas_ghpython wasn't installed, hence the PR.

### What type of change is this?

- [x] New feature in a **backwards-compatible** manner.

### Checklist

1. ~~Add the change to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading: `Added`, `Changed`, `Removed`.~~
(Too small of a change to add to the changelog)
1. [x] Run all tests on your computer (i.e. `invoke test`).

### CODE REVIEW

File: src/compas_rhino/install.py [hunk 1]
```diff
@@ -73,6 +68,14 @@ def install(version=None, packages=None):
 
     if not packages:
         packages = INSTALLABLE_PACKAGES
+        if system == 'darwin' and version == '5.0':
+            packages.remove('compas_ghpython')
+    elif system == 'darwin' and 'compas_ghpython' in packages:
```
Comment (tomvanmele) [inline src/compas_rhino/install.py]: this seems to remove `compas_ghpython` from the packages list for both versions of Rhino for Mac. no?

**Review by tomvanmele — CHANGES_REQUESTED** [review]

File: src/compas_rhino/install.py (hunk 1 above)
Comment (tetov) [reply src/compas_rhino/install.py]: Yes. Should include ’and version == ”5.0”’. Will fix!

Comment (tetov) [reply src/compas_rhino/install.py]: Or do a check once for is_compatible if mac and 5.0 to make it cleaner.

**Comment by tetov** [comment]
```python
if system == 'win32':
        print('Installing COMPAS packages to Rhino {0} IronPython lib:'.format(version))
    elif system == 'darwin':
        print('Installing COMPAS packages to Rhino IronPython lib.')
```

The os shouldn't matter for this print statement, so I removed the check.

**Comment by gonzalocasas** [comment]
looking good! Could you please remove the `.pytest_cache` stuff and add the folder to the `.gitignore` as well?

**Comment by tetov** [comment]
> looking good! Could you please remove the `.pytest_cache` stuff and add the folder to the `.gitignore` as well?
That's embarassing :/. Yeah it's in the pathlib fork already, I'll add it to the same line here as well.

**Pull request merged**